src/Sylius/Component/Payment/Model/PaymentRequest.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Payment\Model;
+
+use Sylius\Component\Resource\Model\TimestampableTrait;
+
+class PaymentRequest implements PaymentRequestInterface
+{
+    use TimestampableTrait;
+
+    protected ?string $hash = null;
+
+    protected ?PaymentMethodInterface $method = null;
+
+    protected ?PaymentInterface $payment = null;
+
+    protected string $state = PaymentRequestInterface::STATE_NEW;
+
+    protected string $type = PaymentRequestInterface::DATA_TYPE_CAPTURE;
```

Comment:
```suggestion
    protected string $action = PaymentRequestInterface::ACTION_CAPTURE;
```
Don't think `type` alone really conveys what it is.
At the very least remove the `DATA` from the consts.

Reply:
I've changed the wording of these fields so many times, and "action" is more appropriate, I agree with you.